ray-operator/main.go

```diff
@@ -92,6 +95,7 @@ func main() {
 	flag.StringVar(&configFile, "config", "", "Path to structured config file. Flags are ignored if config file is set.")
 	flag.BoolVar(&useKubernetesProxy, "use-kubernetes-proxy", false,
 		"Use Kubernetes proxy subresource when connecting to the Ray Head node.")
+	flag.StringVar(&featureGates, "feature-gates", "", "A set of key=value pairs that describe feature gates.")
```

Comment:
Should we mention that the value needs to be "true" or "false"?

https://pkg.go.dev/k8s.io/component-base/featuregate#MutableFeatureGate

```
feature1=true,feature2=false,...
```

Reply:
ack, will update

Reply:
Added an example in the flag description